src/components/App/App.js

```diff
@@ -28,7 +29,9 @@ class App extends Component {
     this.state = {
       searchType: '',
       searchLocation: '',
-      hasSearched: false
+      hasSearched: false,
```

Comment:
As of right now, this is not being used - may need it for routing back to home later but may also end up deleting it if it goes unused.